[2024-09-12T02:59:19.147+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-12T02:59:19.541+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: trino_query_dag_ironmale.insert_data_inc scheduled__2024-09-11T00:00:00+00:00 [queued]>
[2024-09-12T02:59:19.595+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: trino_query_dag_ironmale.insert_data_inc scheduled__2024-09-11T00:00:00+00:00 [queued]>
[2024-09-12T02:59:19.607+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-09-12T02:59:19.681+0000] {taskinstance.py:2330} INFO - Executing <Task(TrinoOperator): insert_data_inc> on 2024-09-11 00:00:00+00:00
[2024-09-12T02:59:19.719+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'trino_query_dag_ironmale', 'insert_data_inc', 'scheduled__2024-09-11T00:00:00+00:00', '--job-id', '155', '--raw', '--subdir', 'DAGS_FOLDER/ironmale.py', '--cfg-path', '/tmp/tmpsmnhs8s3']
[2024-09-12T02:59:19.731+0000] {standard_task_runner.py:91} INFO - Job 155: Subtask insert_data_inc
[2024-09-12T02:59:19.723+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=294) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-12T02:59:19.741+0000] {standard_task_runner.py:64} INFO - Started process 324 to run task
[2024-09-12T02:59:19.906+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-09-12T02:59:20.566+0000] {task_command.py:426} INFO - Running <TaskInstance: trino_query_dag_ironmale.insert_data_inc scheduled__2024-09-11T00:00:00+00:00 [running]> on host 92783cd12528
[2024-09-12T02:59:20.614+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:356 DeprecationWarning: The sql_engine_encoding option in [core] has been moved to the sql_engine_encoding option in [database] - the old setting has been used, but please update your config.
[2024-09-12T02:59:21.003+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/utils/log/secrets_masker.py:84 DeprecationWarning: The sensitive_variable_fields option in [admin] has been moved to the sensitive_var_conn_names option in [core] - the old setting has been used, but please update your config.
[2024-09-12T02:59:21.171+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='trino_query_dag_ironmale' AIRFLOW_CTX_TASK_ID='insert_data_inc' AIRFLOW_CTX_EXECUTION_DATE='2024-09-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-11T00:00:00+00:00'
[2024-09-12T02:59:21.188+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-12T02:59:21.301+0000] {sql.py:282} INFO - Executing: insert into hive.ironmale.soul select * from sqlserver.dbo.soul  where hour = 22
[2024-09-12T02:59:21.410+0000] {base.py:84} INFO - Using connection ID 'my_trino_connection' for task execution.
[2024-09-12T02:59:21.446+0000] {base.py:84} INFO - Using connection ID 'my_trino_connection' for task execution.
[2024-09-12T02:59:21.588+0000] {base.py:84} INFO - Using connection ID 'my_trino_connection' for task execution.
[2024-09-12T02:59:21.623+0000] {sql.py:495} INFO - Running statement: insert into hive.ironmale.soul select * from sqlserver.dbo.soul  where hour = 22, parameters: None
[2024-09-12T02:59:21.900+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-12T02:59:21.908+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/operators/sql.py", line 288, in execute
    output = hook.run(
             ^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py", line 442, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py", line 500, in _run_command
    cur.execute(sql_statement)
  File "/home/airflow/.local/lib/python3.12/site-packages/trino/dbapi.py", line 589, in execute
    self._iterator = iter(self._query.execute())
                          ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/trino/client.py", line 819, in execute
    self._result.rows += self.fetch()
                         ^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/trino/client.py", line 839, in fetch
    status = self._request.process(response)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/trino/client.py", line 611, in process
    raise self._process_error(response["error"], response.get("id"))
trino.exceptions.TrinoQueryError: TrinoQueryError(type=INTERNAL_ERROR, name=SERVER_STARTING_UP, message="Trino server is still initializing", query_id=20240912_025921_00001_bf44p)
[2024-09-12T02:59:22.521+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=trino_query_dag_ironmale, task_id=insert_data_inc, run_id=scheduled__2024-09-11T00:00:00+00:00, execution_date=20240911T000000, start_date=20240912T025919, end_date=20240912T025922
[2024-09-12T02:59:22.636+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 155 for task insert_data_inc (TrinoQueryError(type=INTERNAL_ERROR, name=SERVER_STARTING_UP, message="Trino server is still initializing", query_id=20240912_025921_00001_bf44p); 324)
[2024-09-12T02:59:22.690+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 1
[2024-09-12T02:59:22.719+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py:1297 AirflowProviderDeprecationWarning: Call to deprecated class TrinoOperator. (Please use `airflow.providers.common.sql.operators.sql.SQLExecuteQueryOperator`.)
[2024-09-12T02:59:22.746+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-12T02:59:22.748+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
